.eslintrc

```diff
@@ -10,17 +10,10 @@ rules:
   no-nested-ternary: 2
   no-undefined: 2
   radix: 2
-  space-after-keywords: [2, "always"]
+  keyword-spacing: [2, { "before": true, "after": true}]
```

Comment:
Not that it's a big deal but does it need to be `"before":` rather than just `before`?